src/middlewares/authValidation.js

```diff
@@ -0,0 +1,19 @@
+import { validateSignup } from './validations';
+
+/**
+ * @param {object} req
+ * @param {object} res
+ * @param {object} next
+ * @returns{object} return next() if validations pass
+ */
+const signupValidation = async (req, res, next) => {
+    const { error } = validateSignup(req.body);
+    if (error) {
+        const { details } = error;
+    const messages = details.map((err) => err.message.replace(/['"]/g, '')).join(', ');
+    return res.status(400).json({ status: 400, error: messages });

```

Comment:
same thing here, you can move that status code in `../utils/statusCodes.js` then call it here.